escalation/jira-listener/src/main/java/dev/parodos/jiralistener/model/WebhookEvent.java

```diff
@@ -0,0 +1,19 @@
+package dev.parodos.jiralistener.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.extern.jackson.Jacksonized;
+
+@Data
+@Builder
+@Jacksonized
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class WebhookEvent {
+    private String timestamp;
+    private String webhookEvent;
+    private String issue_event_type_name;
```

Comment:
is it intentionally using snake_case?

Reply:
yes, to avoid JSON property renaming
will review later